README.md

```diff
@@ -0,0 +1,190 @@
+# Charmed Data Integrator Operator
+
+## Overview
+
+This charm allows to retrieve product credentials needed to authenticate (SCRAM) with different kinds of data platform products([MongoDB](https://github.com/canonical/mongodb-operator), [MySQL](https://github.com/canonical/mysql-operator), [PostgreSQL](https://github.com/canonical/postgresql-operator), and [Kafka](https://github.com/canonical/kafka-operator)). 
```

Comment:
**question:** You mention here `SCRAM`. Could you confirm or clarify my understanding, this isn't a specifically `SCRAM` integrator, but a generic entrypoint for getting `username`/`password` from a Data charm, is that right? 

And, it wouldn't matter what authentication mechanism the product uses.

If so, I would remove the reference to `SCRAM` here.

Reply:
I totally agree